src/app/pages/Game.tsx

```diff
@@ -193,6 +214,7 @@ export const Game = () => {
 					onUpdate={setRobotMovements}
 					key={JSON.stringify(robotMovements)}
 					teamColor={teamColor(selectedTeam)}
+					readyToPlay={readyToMove}
```

Comment:
I'd hide the entire form if the team is ready to play. Requires less code in the form component.